models/prices/aptos/prices_aptos_tokens.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(
+        schema='prices_aptos',
+        alias = 'tokens',
+        materialized='table',
+        file_format = 'delta',
+        tags = ['static', 'dunesql']
+        )
+}}
+SELECT 
+    token_id
+    , blockchain
+    , symbol
+    , from_base58(contract_address) as contract_address
+    , decimals
+FROM
+(
+    VALUES
+    ('apt-aptos','aptos','APT','0x1::aptos_coin::AptosCoin',8)
```

Comment:
Do you have any official explorer link for this token?

Reply:
https://aptoscan.com/coin/0x1::aptos_coin::AptosCoin